src/ll.Translator.js

```diff
@@ -21,6 +21,10 @@ OO.initClass( ll.Translator );
 
 /* Static methods */
 
+ll.Translator.static.outerSeparator = ':!!!:';
```

Comment:
It would be nice to document what these are and where they're used, especially since unbundle() relies on these separators not being meaningful / needing escaping in regexes (or alternatively, we could modify unbundle() to regex-escape the separators)

Reply:
To be honest they're probably a bit Apertium-specific – ultimately they probably shouldn't be in the base class.